Signal/src/view controllers/PrivacySettingsTableViewController.m

```diff
@@ -106,6 +135,10 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
             return 1;
         case PrivacySettingsTableViewControllerSectionIndexCalling:
             return 1;
+        case PrivacySettingsTableViewControllerSectionIndexCallKitEnabled:
+            return self.supportsCallKit ? 1 : 0;
+        case PrivacySettingsTableViewControllerSectionIndexCallKitPrivacy:
+            return (self.supportsCallKit && [[Environment getCurrent].preferences isCallKitEnabled]) ? 1 : 0;
```

Comment:
It seems like this should hide the callkit privacy section when callkit is disabled. and yet...
<img src="https://cloud.githubusercontent.com/assets/217057/23327900/b02e30f8-fac9-11e6-925e-b01024130d64.jpg" width=400 />